src/test/java/net/datafaker/providers/base/GarmentSizeTest.java

```diff
@@ -1,14 +1,25 @@
 package net.datafaker.providers.base;
 
 
+import java.util.List;
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class GarmentSizeTest extends BaseFakerTest<BaseFaker> {
 
     @Test
-    void size() {
-        assertThat(faker.garmentSize().size()).isNotEmpty();
+    void testSize() {
+        // Given
+        List<String> actualSizes = getActualSizes();
```

Comment:
I had considered doing this as:
```java
List<String> actualSizes = List.of("XS", "S", "M", "L", "XL", "XXL", "XXXL");
```

Since it's likely always a small set of values, but I kept waffling back and forth. So let me know which form you like better.

Reply:
Oh I just realized that List.of wouldn’t have been good for your Java 8 support. Still coulda declared it more inline’ishly.